conformance/lang/expressions/field-access-expression/field_access_expr_for_lax_type.balt

```diff
@@ -0,0 +1,1093 @@
+    json x6 = 12;
+    json x7 = "JSON";
+    json x8 = ();
+    json[] x9 = [x1, x2, x3];
+    json x10 = {x1, x9};
+    json x11 = {x1, x2, x3, x4, x5, x6, x7, x8, readonly x9, x10};
+
+    io:println(x11.x1); // @output 2
+    io:println(x11.x2); // @output 1.112
+    io:println(x11.x3); // @output 211.1
+    io:println(x11.x4); // @output string
+    io:println(x11.x5); // @output true
+    io:println(x11.x6); // @output 12
+    io:println(x11.x7); // @output JSON
+    io:println(x11.x8); // @output
+    io:println(x11.x9); // @output [2,1.112,211.1]
+    io:println(x11.x10); // @output {"x1":2,"x9":[2,1.112,211.1]}
+    io:println(x11.x10.x1); // @output 2
+    io:println(x11.x10.x9); // @output [2,1.112,211.1]
+}
+
+Test-Case: output
+Description: Test field access expression by accessing existing fields when they belong to JSON objects with valid
+             field-names.
+Labels: array-type, boolean, decimal, DecimalFloatingPointNumber, DecimalNumber, field-access-expr, float,
+        FloatingPointTypeSuffix, HexFloatingPointLiteral, HexIntLiteral, json, list-constructor-expr,
+        mapping-constructor-expr, nil-literal, readonly-type, string
+
+function init() {
+    json x1 = {a: 2, z: .1, A: "string1", Z: false, _1: 21.3d, _a: 0x12, _A: 0x.21, _12: true, __: (), සිංහල: "Sinhala"};
+    json x2 = {\1y: 121, \_: 1, \u{a}: "string2", \u{12e}: "string3", _\_: 1, a\u{a}: "string2", _\u{12e}: "string3"};
+    json x3 = {'02: 2, '_1: .192, '\1: 1, '\u{ab1}: null, 'int: 12};
+    json x4 = {__: x1, \.: x2};
+
+    io:println(x1.a); // @output 2
+    io:println(x1.z); // @output 0.1
+    io:println(x1.A); // @output string1
+    io:println(x1.Z); // @output false
+    io:println(x1._1); // @output 21.3
+    io:println(x1._a); // @output 18
+    io:println(x1._A); // @output 0.12890625
+    io:println(x1._12); // @output true
+    io:println(x1.__); // @output
+    io:println(x1.සිංහල); // @output Sinhala
+
+    io:println(x2.\1y); // @output 121
+    io:println(x2.\_); // @output 1
+    io:println(x2.\u{a}); // @output string2
+    io:println(x2.\u{12e}); // @output string3
+    io:println(x2._\_); // @output 1
+    io:println(x2.a\u{a}); // @output string2
+    io:println(x2._\u{12e}); // @output string3
+
+    io:println(x3.'02); // @output 2
+    io:println(x3.'_1); // @output 0.192
+    io:println(x3.'\1); // @output 1
+    io:println(x3.'\u{ab1}); // @output
+    io:println(x3.'int); // @output 12
+
+    io:println(x4.__.__); // @output
+    io:println(x4.\..__); // @output 1
+}
+
+Test-Case: output
+Description: Test field access expression by accessing existing fields when they belong to JSON objects. Here, mapping
+             constructor contains computed-name-field. Test case 1.
+Labels: array-type, computed-name-field, DecimalNumber, error-type, field-access-expr, if-else-stmt, is-expr, json,
+        list-constructor-expr, logical-negation, mapping-constructor-expr, nil-literal, string, value:toBalString
+
```

Comment:
`function-defn`, `functon-call-expr` labels are missing.

Reply:
Added `null`. Here function-call-expr is not tested explicitly and it belongs to minimal subset. Therefore, we can ignore these labels.